cmd/fluxd/main.go

```diff
@@ -561,7 +560,7 @@ func main() {
 	// Registry components
 	var imageRegistry registry.Registry = registry.ImageScanDisabledRegistry{}
 	var cacheWarmer *cache.Warmer
-	if *registryScanning {
+	if !*registryDisableScanning {
```

Comment:
Ah! Here is a double-negative, and it does make one look twice -- but I am less concerned about this one than the UI itself, having brought it up ..